src/__test__/kitchensink.test.js

```diff
@@ -9,9 +9,11 @@ import {noop} from '../noop'
 import {render} from '../render'
 import {renderInDocument} from './support/renderInDocument'
 
+import 'rxjs/add/observable/from'
 import 'rxjs/add/observable/interval'
 import 'rxjs/add/operator/map'
 import 'rxjs/add/operator/merge'
+import 'rxjs/add/observable/of'
```

Comment:
They use to, but the code that used it was moved to the regression test file. It might be useful to keep them as they are such a common tool to use with Rx anyway.